<launch>
  <!-- size of map, change the size in x, y, z according to your application -->
  <arg name="map_size_x" value="40.0"/>
  <arg name="map_size_y" value="20.0"/>
  <arg name="map_size_z" value=" 5.0"/>

  <!-- topic of your odometry such as VIO or LIO -->
  <arg name="odom_topic" value="/state_ukf/odom" />

  <node pkg="rl_control_test" name="api_demo" type='api_demo.py' output="screen">
  <remap from="/so3_cmd" to="so3_cmd"/>
  <remap from="/quadrotor_odom" to="$(arg odom_topic)"/>
  <remap from="/quadrotor_cloud" to="/pcl_render_node/cloud"/>
  <remap from="/quadrotor_imu" to="/quadrotor_simulator_so3/imu"/>
  </node>

</launch>
